app/models/labware_creators/plate_split_to_tube_racks.rb

```diff
@@ -72,6 +72,14 @@ class PlateSplitToTubeRacks < Base
     PARENT_PLATE_INCLUDES =
       'wells.aliquots,wells.aliquots.sample,wells.downstream_tubes,wells.downstream_tubes.custom_metadatum_collection'
 
+    def validate_file_presence
+      if contingency_file.blank? && sequencing_file.blank?
```

Comment:
Can this whole if else statement not be simplified to a check for sequencing_file.blank?
Error 'Sequencing file must be present'.

Reply:
Yeah make sense. Thanks.